web/src/package.json

```diff
@@ -89,6 +89,7 @@
     "moment": "^2.22.2",
     "react": "16.8.6",
     "react-apollo": "2.5.6",
+    "react-apollo-hooks": "0.4.5",
```

Comment:
Should we reconsider using the apollo beta?

This package recommends using that for new work, as it is planned to be deprecated. https://github.com/trojanowski/react-apollo-hooks

Reply:
👍